test/evergreen.yml

```diff
@@ -4739,6 +4739,50 @@ tasks:
           test_name: "log"
       - func: "aggregate-perf-stats" 
 
+  - name: aggregate-perf-stats-macos
+    depends_on:
+      - name: perf-test-medium-btree
+      - name: perf-test-medium-btree-backup
+      - name: perf-test-mongodb-oplog
+      - name: perf-test-mongodb-small-oplog
+      - name: perf-test-mongodb-secondary-apply
+      - name: perf-test-evict-btree
+      - name: perf-test-evict-btree-1
+      - name: perf-test-log
+    commands:
+      - func: "get project"
+      - command: shell.exec
+        params:
+          shell: bash
+          script: |
+            mkdir perf_stats
+      - func: "fetch perf stats"  
+        vars:
+          test_name: "medium-btree" 
+      - func: "fetch perf stats" 
+        vars:
+          test_name: "medium-btree-backup" 
+      - func: "fetch perf stats" 
+        vars:
+          test_name: "mongodb-oplog" 
+      - func: "fetch perf stats" 
+        vars:
+          test_name: "mongodb-small-oplog" 
+      # FIXME-BUILD-16428: Re-enable once snappy is installed on mac platforms.
```

Comment:
Instead of the comment here could we stick a comment at the top of the job saying something along the lines of "When BUILD-16428 is fixed we can delete this task in favour of aggregate-perf-tests which also runs mongodb-secondary-apply"?

Reply:
Done, [6c349a4](https://github.com/wiredtiger/wiredtiger/pull/8525/commits/6c349a4fd0ec741744132000d8acf38c39733320)